bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -186,6 +193,7 @@
 		</supported-bridge-type-refs>
 		<label>@text/energyMeterLiveLabel</label>
 		<description>@text/energyMeterLiveDescription</description>
+		<semantic-equipment-tag>ElectricMeter</semantic-equipment-tag>
```

Comment:
This terminology feels strange to me. I would have expected ElectricityMeter. An electricity meter measures electricity. An electric meter uses electricity to measure something.

Reply:
;)